.github/workflows/autorebase.yml

```diff
@@ -0,0 +1,19 @@
+name: Automatic Rebase
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase') && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
```

Comment:
@cortinico would it be ok to also allow the author of the PR to use this command?

Reply:
Please do 🙏 The command is currently broken actually as this condition is too strict. Feel free to tweak it